blog/_posts/2020-02-07-delete-clusters-faster-with-kind.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: delete-clusters-faster-with-kind
+date: 2020-02-07
+author: nick
+layout: blog
+title: "Kill Your Kubernetes Clusters Faster with Kind"
+subtitle: "Announcing a Big Improvement in Local Development"
+tags:
+  - tilt
+keywords:
+  - tilt
+  - kind
+  - local development
+---
+
+TK Selena Gomez Joke
+
+At the ClusterAPI meeting in January
+
+Contributors create a Kind cluster and use Tilt for interactive development
```

Comment:
maybe (here or elsewhere) a bit about WHY they like kind -- encouraging fresh cluster, etc.

also maybe worth pointing out that they liked it so much that they even used it when it was slow as heck (`kind load`), so _kind-but-fast_ is a gamechanger

Reply:
ya! will expand on this.